Ch07/Ch07.md

```diff
@@ -340,11 +340,11 @@ The only important distinction between a rehab node and a promotable node is wha
 relevant in pathological cascading failure scenarios.
 Consider a cluster of three nodes with a database residing on nodes A and B. First, we have node B failing for some reason. (Perhaps the operations team took it
 down to apply patches.) The supervisor will detect that and move node B to the rehab group. Node A will assume all duties that were previously assigned to 
-node B. 
+node B. You can see this in Figure 7.3, the red `R` icon for node B indicates that it is in rehab mode.
```

Comment:
You can see this in Figure 7.3; the red 'R' icon for node B indicates that it is in rehab mode.